index.js

```diff
@@ -638,7 +638,7 @@ RedisClient.prototype.return_reply = function (reply) {
                 reply = reply_to_object(reply);
             }
 
-            try_callback(command_obj.callback, reply);
+            command_obj.callback(null, reply);
```

Comment:
The try callback is actually preventing us from emitting the error properly, so this has to be removed for it.